### ISSUE TITLE
Password Help

### ISSUE THREAD

**Issue by cmetz7** [jchoca/fishbowl-api#1]
self.password = hashlib.md5(parrword).digest().encode('base64').replace('\n', "") is giving me the error:
TypeError: Unicode-objects must be encoded before hashing. I've tried encoding my password, but that is telling me: AttributeError: 'bytes' object has no attribute 'encode'

**Comment by jchoca** [comment]
What version of Python are you using?  I think I was using 2.7 when I wrote this.

**Comment by cmetz7** [comment]
I am using 3.4. I think I may have found a work around, but I am not sure. I changed the **init**: self.password to be = password. Then down in the get_response() method I added a decode to byte = self.stream.recv(1).decode(‘utf-8’). That seemed to work since it didn’t return any errors and I have my test servers address and port in there. I know there are some differences between 2.x and 3.x on how the encoding works. I was reading online that I might want to try base64.b64encode().
